### ISSUE TITLE
emulator crashes when i stop recording video -> "allocator 3.x is not supported"

### ISSUE THREAD

**Issue by durannapdann** [flutter/flutter#63952]
this only happens on api 30 'versions ..

flutter run --verbose -> [error.txt](https://github.com/flutter/flutter/files/5084256/error.txt)

code ->  [code.txt](https://github.com/flutter/flutter/files/5084260/code.txt)

pubspec.yaml ->
  flutter_better_camera: ^0.6.1
  path_provider: ^1.6.11
  video_player: ^0.10.11+2
  permission_handler: ^5.0.1+1
  image_gallery_saver: '^1.5.0'
  path:

**Comment by darshankawar** [comment]
Hi @durannapdann,
I tried your code on emulator with api 30 on latest Stable (1.20.2) and was able to record a video properly without any crash.

[63952.mov.zip](https://github.com/flutter/flutter/files/5089137/63952.mov.zip)


<details>
<summary>flutter doctor -v</summary>

```
Performing hot restart...
Syncing files to device sdk gphone x86 arm...
I/OMXClient( 5595): IOmx service obtained
I/ACodec  ( 5595): codec does not support config priority (err -2147483648)
Restarted application in 13,015ms.
I/OMXClient( 5595): IOmx service obtained
I/ACodec  ( 5595): codec does not support config priority (err -2147483648)
I/OMXClient( 5595): IOmx service obtained
I/ACodec  ( 5595): codec does not support config priority (err -2147483648)
I/ple.hello_worl( 5595): Background young concurrent copying GC freed 117821(4239KB) AllocSpace objects, 27(1836KB) LOS objects, 18% free, 24MB/29MB, paused 28us total 216.050ms
I/OMXClient( 5595): IOmx service obtained
I/ACodec  ( 5595): codec does not support config priority (err -2147483648)
I/OMXClient( 5595): IOmx service obtained
I/ACodec  ( 5595): codec does not support config priority (err -2147483648)
I/OMXClient( 5595): IOmx service obtained
I/ACodec  ( 5595): codec does not support config priority (err -2147483648)
W/ple.hello_worl( 5595): Long monitor contention with owner main (5595) at void android.hardware.camera2.impl.CameraDeviceImpl.waitUntilIdle()(CameraDeviceImpl.java:1287) waiters=0 in void android.hardware.camera2.impl.CameraDeviceImpl$CameraDeviceCallbacks.onResultReceived(android.hardware.camera2.impl.CameraMetadataNative, android.hardware.camera2.impl.CaptureResultExtras, android.hardware.camera2.impl.PhysicalCaptureResultInfo[]) for 159ms
I/OMXClient( 5595): IOmx service obtained
I/ACodec  ( 5595): codec does not support config priority (err -2147483648)
I/ExoPlayerImpl( 5595): Release f0c13c3 [ExoPlayerLib/2.9.6] [generic_x86_arm, sdk_gphone_x86_arm, Google, 30] [goog.exo.core]
I/flutter ( 5595): file:///storage/emulated/0/hello_world/1597738954865.mp4
D/SurfaceUtils( 5595): disconnecting from surface 0xeaa1c578, reason disconnectFromSurface
I/ExoPlayerImpl( 5595): Init 2e87551 [ExoPlayerLib/2.9.6] [generic_x86_arm, sdk_gphone_x86_arm, Google, 30]
I/OMXClient( 5595): IOmx service obtained
D/SurfaceUtils( 5595): connecting to surface 0xeaa064d8, reason connectToSurface
I/MediaCodec( 5595): [OMX.google.h264.decoder] setting surface generation to 5729283
D/SurfaceUtils( 5595): disconnecting from surface 0xeaa064d8, reason connectToSurface(reconnect)
D/SurfaceUtils( 5595): connecting to surface 0xeaa064d8, reason connectToSurface(reconnect)
E/ACodec  ( 5595): [OMX.google.h264.decoder] setPortMode on output to DynamicANWBuffer failed w/ err -1010
I/ACodec  ( 5595): codec does not support config priority (err -1010)
I/OMXClient( 5595): IOmx service obtained
I/ACodec  ( 5595): codec does not support config priority (err -2147483648)
I/ple.hello_worl( 5595): Background young concurrent copying GC freed 60100(2942KB) AllocSpace objects, 31(2108KB) LOS objects, 14% free, 25MB/29MB, paused 92us total 422.313ms
W/System  ( 5595): A resource failed to call release. 
I/chatty  ( 5595): uid=10152(com.example.hello_world) FinalizerDaemon identical 1 line
W/System  ( 5595): A resource failed to call release. 
D/MediaCodec( 5595): [OMX.google.h264.decoder] setting dataspace on output surface to #104
D/SoftwareRenderer( 5595): setting dataspace on output surface to #104
D/AudioTrack( 5595): getTimestamp_l(20): device stall time corrected using current time 2468322124900
I/ple.hello_worl( 5595): Background concurrent copying GC freed 33118(3549KB) AllocSpace objects, 258(17MB) LOS objects, 49% free, 12MB/25MB, paused 101us total 2.401s
W/System  ( 5595): A resource failed to call release. 
I/OMXClient( 5595): IOmx service obtained
I/ACodec  ( 5595): codec does not support config priority (err -2147483648)
```

</details>

When do you exactly get a crash ? What version are you on ?

**Comment by TahaTesser** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.
